feat(submit project work): submit project for review

ffurzy opened the pull request.

ffurzy committed:
feat(submit project work): submit project for review

yana-yanchenko reviewed (commented):
Необходимо развернуть общий комментарий ⬇️

Добрый день! Проект выполнен на отличном уровне!

Отлично:

- Все комментарии с отметкой "Отлично" в коде
- Хорошо реализовали форму из предоставленных компонентов
- Хорошая логика передачи данных от `App` в форму и обратно
- Отлично, что нет типов `any`
- Отлично типизировали проект


Надо исправить:

- Все пункты, отмеченные комментарием "Надо исправить"
- [isOpen, setIsOpen] нужно назвать [isMenuOpen, setIsMenuOpen], чтобы было понятно, что тут открываете именно меню
- По заданию нужно использовать библиотеку clsx для объединения 2х и более стилей

Можно лучше:

- Обратите внимание на все комментарии с отметкой "Можно лучше" в коде
- Лучше выделить компонент `App` в отдельный файл, чтобы не смешивать все в `index.js`
- `clsx` можно не использовать, если только **один** селектор передается


Обязательно внесите все исправления по пунктам "надо исправить". Улучшения по разделу "можно лучше" - на ваше усмотрение, но они помогут повысить качество кода.

Успешного рефакторинга!

ffurzy committed:
fix: fix(article-params-form): apply review changes (rename state, submit on form, clsx usage)

fix(article-params-form): apply review changes (rename state, submit on form, clsx usage)

yana-yanchenko reviewed (commented):
Добрый день! Ваши навыки растут с каждым проектом. Не останавливайтесь на достигнутом!

Работа принята!
